Source: @pullmoll
Upstream: no
Reason: In yacc mode %pure-parser emits a warning to use %define api.pure instead

--- a/libsrc/Wi/nquad_p.y	2018-08-16 01:06:21.000000000 +0200
+++ b/libsrc/Wi/nquad_p.y	2020-01-27 15:58:33.086194051 +0100
@@ -21,7 +21,7 @@
  *
  */
 
-%pure-parser
+%define api.pure
 %parse-param {ttlp_t * ttlp_arg}
 %parse-param {yyscan_t yyscanner}
 %lex-param {ttlp_t * ttlp_arg}
--- a/libsrc/Wi/sparql_p.y	2018-08-16 01:06:21.000000000 +0200
+++ b/libsrc/Wi/sparql_p.y	2020-01-27 15:58:32.947194056 +0100
@@ -36,7 +36,7 @@
 
 Whitespaces in all other places, including two whitespaces after "::=" in BNF comments */
 
-%pure-parser
+%define api.pure
 %parse-param {sparp_t * sparp_arg}
 %lex-param {sparp_t * sparp_arg}
 %expect 14
--- a/libsrc/Wi/sql3.y	2018-08-16 01:06:21.000000000 +0200
+++ b/libsrc/Wi/sql3.y	2020-01-27 15:58:33.013194054 +0100
@@ -25,7 +25,7 @@
  *
  */
 
-%pure-parser
+%define api.pure
 %parse-param {yyscan_t scanner}
 %lex-param {yyscan_t scanner}
 /*%parse-param {sql_comp_context_t* scs_arg}*/
--- a/libsrc/Wi/turtle_p.y	2018-08-16 01:06:21.000000000 +0200
+++ b/libsrc/Wi/turtle_p.y	2020-01-27 15:58:33.083194051 +0100
@@ -21,7 +21,7 @@
  *
  */
 
-%pure-parser
+%define api.pure
 %parse-param {ttlp_t * ttlp_arg}
 %parse-param {yyscan_t yyscanner}
 %lex-param {ttlp_t * ttlp_arg}
--- a/libsrc/Wi/xpathp.y	2018-08-16 01:06:21.000000000 +0200
+++ b/libsrc/Wi/xpathp.y	2020-01-27 15:58:33.086194051 +0100
@@ -25,7 +25,7 @@
  *
  */
 
-%pure-parser
+%define api.pure
 %parse-param {xpp_t * xpp_arg}
 %lex-param {xpp_t * xpp_arg}
 
